src/appmixer/google/gmail/FindEmails/component.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "appmixer.google.gmail.FindEmails",
     "author": "Appmixer <info@appmixer.com>",
-    "description": "Finds emails in Gmail based on a search query. You can retrieve emails as a single item, multiple items, or save them directly to a file. Each message resource contains only the Email ID and Thread ID. To get detailed information about specific emails, use the Get Email component. For guidance on search queries, refer to the <a href=\"https://support.google.com/mail/answer/7190?hl=en\" target=\"_blank\">Gmail Search Documentation</a>.",
+    "description": "Finds emails in Gmail based on a search query. You can retrieve emails as a single item, multiple items, or save them directly to a file. Each message resource contains only the Email ID and Thread ID. To get detailed information about specific emails, use the Get Email component. For guidance on search queries, refer to the <a href=\"https://support.google.com/mail/answer/7190?hl=en\" target=\"_blank\">Gmail Search Documentation</a>. This Component will return 500 records at Maximum",
```

Comment:
instead of the "This Component will return 500 records at Maximum" 

"This component can return a maximum of 500 records"  or "This component has an upper limit of 500 records it can return"

not sure abut the capital C an M